sdk/core/core-lro/src/utils/constants.ts

```diff
@@ -0,0 +1,13 @@
+export type LongRunningOperationStates =
+  | "InProgress"
+  | "Succeeded"
+  | "Failed"
+  | "Canceled"
+  | "Cancelled";
```

Comment:
having both "Canceled" and "Cancelled" is very odd

Reply:
Are some services British and some American?